dev/plugins/hu.elte.txtuml.diagnostics/src/hu/elte/txtuml/diagnostics/session/DiagnosticsPlugin.java

```diff
@@ -41,11 +45,29 @@ public DiagnosticsPlugin(int diagnosticsPort, String projectName, String working
 		modelMapper = new ModelMapper(projectName);
 		instanceRegister = new InstanceRegister();
 		animator = new Animator(instanceRegister, modelMapper);
+		server = new DiagnosticsServer(this);
+		
+		try {
+			server.start(httpPort);
+		} catch (IOException e) {
+			Logger.sys.error("Couldn't start HTTP server on port " + httpPort + " in service instance 0x", e);
+			return;
+		}
+		Logger.sys.info("txtUML diagnostics connection is set on HTTP port " + httpPort );
```

Comment:
This will never be seen by the user, honestly don't know where this is redirected to.

Reply:
Not necessary to remove, just to clarify.